[{"_time":1653662109,"cribl_breaker":"Break on newlines","_raw":"05/27/2022 08:05:09 PM\nLogName=Application\nSourceName=ShadowProtectSvc\nEventCode=1121\nEventType=2\nType=Error\nComputerName=myhost.acme.com\nUser=NOT_TRANSLATED\nSid=S-1-5-18\nSidType=0\nTaskCategory=The operation completed successfully.\nOpCode=None\nRecordNumber=28535917\nKeywords=Classic\nMessage=Backup status: failed\r\nImage file: \\\\ab-back01\\Backups\\Servers\\SY-FS01\\MYHOST_C_VOL\r\nLog file: C:\\Program Files (x86)\\StorageCraft\\ShadowProtect\\Logs\\{98BFAA1C-6D88-4DB5-9EE9-DBCBA9F0123F}.txt\r\nStart time: 27/05/2022 8:05:00 PM\r\nModule: service\r\nCode: 509\r\nMessage: Cannot get access to destination object"}]